linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_usergroup_modification/tests/auditctl_wrong_value.fail.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# packages = audit
+
+# use auditctl
+sed -i "s%^ExecStartPost=.*%ExecStartPost=-/sbin/auditctl%" /usr/lib/systemd/system/auditd.service
+
+echo "-w /etc/group -p w -k audit_rules_usergroup_modification" >> /etc/audit/audit.rules
```

Comment:
I recommend adding a test scenario where the permissions are wrong and the key is missing.

Reply:
done